### ISSUE TITLE
Mixer app does not connect to device

### ISSUE THREAD

**Issue by Danix76** [onkelDead/tascam-gtk#4]
Hi
Thanks so much for the driver, it works great! The GTK interface, I can compile and it will load however, the controls do not attach to the device. I'm running debian with kernel 4.9 amd 64... i get this from the mixer when i move faders:
Control name='Line Volume',index=0 element not found
Control name='Line Volume',index=0 element not found
Control name='Line Volume',index=0 element not found
Control name='Line Volume',index=0 element not found
Control name='Line Volume',index=0 element not found
Control name='Line Volume',index=0 element not found
Control name='Level Meter',index=0 element not found

dmesg:
   16.202857] usb 3-4: new high-speed USB device number 4 using xhci_hcd
[   16.343625] usb 3-4: New USB device found, idVendor=0644, idProduct=8047
[   16.343630] usb 3-4: New USB device strings: Mfr=1, Product=2, SerialNumber=3
[   16.343632] usb 3-4: Product: US-16x08
[   16.343635] usb 3-4: Manufacturer: TASCAM
[   16.343637] usb 3-4: SerialNumber: no serial number
[   17.739774] snd_usb_audio: loading out-of-tree module taints kernel.
[   17.985569] usbcore: registered new interface driver snd-usb-audio

gnome-mixer works fine, so does xfce4 mixer. I'm running the xfce4 interface.

I tried recompiling the GUI mixer twice but, it still won't connect to the interface.

The LV2 plugin causes Ardour 5.8 to crash.
The QT5 interface seems to be incomplete.

If I can help in any way to get the GTK interface working, please let me know.

Thanks for your hard work and, I hope you have time to look at this simple fix since obviously, it works on your system :)

Thanks again friend!

Dan Murray.

**Comment by onkelDead** [comment]
Hi,
I was working on a integration of the driver patch in the mainline kernel. It's included in kernel 4.12.
That's why tascam-gtk and the kernel driver patch on github did ran out of synch.

I've updated the custom kernel driver patch on github right now.

BTW:
The QT5 mixer is rather poor, outdated and wont be maintained any longer. I'll remove this repo soon.
The lv2 plugin needs a bit more attention. I'll inform you once I fixed it.

Greets
Detlef
